chainer/link.py

```diff
@@ -426,6 +426,39 @@ def addgrads(self, link):
         for name in self._params:
             dst[name].addgrad(src[name])
 
+    def enable_update(self):
+        """Enables update rules of all parameters under the link hierarchy.
+
+        This method sets the :attr:`~chainer.UpdateRule.enabled` flag of the
+        update rule of each parameter variable to ``True``.
+
+        """
+        for param in self.params():
+            rule = param.update_rule
+            if rule is not None:
```

Comment:
This is already tested by `test_enable_update` (note that parameters `y` and `v` do not have update rules).